### DISCUSSION TITLE
Error while i try to install more on code window screen (im a newbie)

### DISCUSSION THREAD

**Discussion by savagecodess** [lllyasviel/Fooocus#3668] [Q&A]
so this is how it goes 
C:\Users\CM Laptops\Desktop>.\python_embeded\python.exe -s Fooocus\entry_with_update.py --preset realistic
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py', '--preset', 'realistic']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.5.5
Loaded preset: C:\Users\CM Laptops\Desktop\Fooocus\presets\realistic.json
[Cleanup] Attempting to delete content of temp dir C:\Users\CMLAPT~1\AppData\Local\Temp\fooocus
[Cleanup] Cleanup successful
Downloading: "https://huggingface.co/mashb1t/misc/resolve/main/xl-to-v1_interposer-v4.0.safetensors" to C:\Users\CM Laptops\Desktop\Fooocus\models\vae_approx\xl-to-v1_interposer-v4.0.safetensors

Key already registered with the same priority: C10

C:\Users\CM Laptops\Desktop>pause
Press any key to continue . . .
Terminate batch job (Y/N)?

**Comment by savagecodess** [comment]
really need help..

**Reply by Rogala** [reply]
Delete the config.txt file, and just run run.bat
